pcdswidgets/vacuum/base.py

```diff
@@ -441,8 +454,62 @@ def _handle_icon_click(self):
         self._expert_display = display
         display.destroyed.connect(self._cleanup_expert_display)
 
-        if display:
-            display.show()
+        if len(self.ui_file_path) > 0:
+            self.tabWidget = QTabWidget()
+            self.embeddedDisplay = list()
+            self.tabWidget.setTabPosition(2)
```

Comment:
When working with enums it's usually best practice to reference the enum name instead of the value to make the code more readable.
```suggestion
            self.tabWidget.setTabPosition(QTabWidget.TabPosition.West)
```